ci/build-mingw64.sh

```diff
@@ -310,8 +313,10 @@ if [ "$2" = pack ]; then
         libgcc_*.dll lib{ssp,stdc++,winpthread}-[0-9]*.dll # compiler runtime
         av*.dll sw*.dll lib{ass,freetype,fribidi,harfbuzz,iconv,placebo}-[0-9]*.dll
         lib{shaderc_shared,spirv-cross-c-shared,dav1d}.dll zlib1.dll
-        # note: vulkan-1.dll is not here since drivers provide it
     )
+    if [[ "$TARGET" != "i686-"* && -f vulkan-1.dll ]]; then
```

Comment:
the point was that you don't need the i686 check here, vulkan is only built on x86_64